evennia/server/inputfuncs.py

```diff
@@ -41,6 +42,28 @@
 def _NA(o):
     return "N/A"
 
+def _strip_unsafe(session, txt):
```

Comment:
If this is intended to be overridden, it's best to leave it named without the leading `_`. We have not been very consistent with this everywhere, but overall, "_private" methods tends to be something that is not really a part of the public API, and which can change more willy-nilly (this is the use in e.g. Twisted and Django too).